fairseq/modules/multihead_attention.py

```diff
@@ -96,6 +102,29 @@ def forward(self, query, key, value, key_padding_mask=None, incremental_state=No
         assert embed_dim == self.embed_dim
         assert list(query.size()) == [tgt_len, bsz, embed_dim]
 
+        if self.enable_torch_version and not self.onnx_trace and incremental_state is None and not static_kv:
+            if self.qkv_same_dim:
+                return  F.multi_head_attention_forward(query, key, value, 
+                                                       self.embed_dim, self.num_heads,
+                                                       self.in_proj_weight,
+                                                       self.in_proj_bias, self.bias_k, self.bias_v,
+                                                       self.add_zero_attn, self.dropout,
+                                                       self.out_proj.weight, self.out_proj.bias,
+                                                       self.training, key_padding_mask, need_weights,
+                                                       attn_mask)
+            else:
+                return  F.multi_head_attention_forward(query, key, value, 
+                                                       self.embed_dim, self.num_heads,
+                                                       torch.empty([0]),
+                                                       self.in_proj_bias, self.bias_k, self.bias_v,
+                                                       self.add_zero_attn, self.dropout,
+                                                       self.out_proj.weight, self.out_proj.bias,
+                                                       self.training, key_padding_mask, need_weights,
+                                                       attn_mask, use_separate_proj_weight=True,
```

Comment:
use_separate_proj_weight seems to be undefined in the latest nightly build :/

Reply:
we have an open PR which needs be landed asap. https://github.com/pytorch/pytorch/pull/21288